diesel/src/r2d2.rs

```diff
@@ -1,4 +1,6 @@
-//! Connection pooling via r2d2
+//! Connection pooling via r2d2.
+//! Note that in order to use this module, you need to have Diesel
+//! installed with the `r2d2` feature
```

Comment:
I don't like the use of the word "installed" for this. What do you think of "Note: This module requires enabling the `r2d2` feature"

Reply:
You're right. That's a lot better.